### PR TITLE
Feat: Added image and input component and made responsive to some extent

### PR DESCRIPTION
**Pull request by sainath7878**
Added image and input component. Made responsive to some extent. Please review and let me know if any changes are required.

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
LGTM, great work!

**Comment by partha8** [comment]
suggestion, keep overflow: auto for side-nav

**Review by Dev-AkshatGupta — APPROVED** [review]
LGTM buddy Great work

File: documentation.css [hunk 1]
```diff
@@ -36,17 +39,17 @@
...
-    transition: 0.3s;
 }
 /* ------- header end -------  */
 
 main{
     display: grid;
-    grid-template-columns: 22% 1fr 5%;
+    grid-template-columns: 25% 1fr 5%;
     grid-template-areas: 
     "side-nav documentation ...."
```
Comment (venkateshwarans) [inline documentation.css:54]: I like what you have done here to get space on the right of the viewport. Any reason for not going with padding?

Comment (sainath7878) [reply documentation.css:54]: No specific reason, I felt this is an easy way. Is this not suggested?

Comment (venkateshwarans) [reply documentation.css:54]: I am not sure either. it practically achieves the purpose. I think it won't be an issue when making the site responsive either. Maybe ask someone in the server if they can opine.

**Review by BalajiHarinath — APPROVED** [review]
Looks good to me

**Review by venkateshwarans — APPROVED** [review]

**Pull request merged**